### DISCUSSION TITLE
Branch Names Truncated

### DISCUSSION THREAD

**Discussion by NWEveri** [JanDeDobbeleer/oh-my-posh#4833] [Troubleshoot] [answered]
I'm not sure if this is the right place to ask this question, but I have an issue with branch names being truncated at 25 characters, despite having $GitPromptSetting.BranchNameLimit set to 0. Is there another config setting I need to alter?
See my screenshot below.
Thanks in advance for any help.
![image](https://github.com/JanDeDobbeleer/oh-my-posh/assets/166707752/2ffc30c2-aa72-427a-8421-62be6a8791b7)

**Comment by JanDeDobbeleer** [comment, marked as answer]
This is regulated by `branch_max_length` in the configuration as well.
